src/dispatch/static/dispatch/src/app/store.js

```diff
@@ -8,12 +8,16 @@ const getDefaultRefreshState = () => {
   }
 }
 
+const latest_commit_hash = import.meta.env.VITE_DISPATCH_COMMIT_MESSAGE
+const latest_commit_message = import.meta.env.VITE_DISPATCH_COMMIT_MESSAGE
+
 const state = {
   toggleDrawer: true,
   refresh: {
     ...getDefaultRefreshState(),
   },
   loading: false,
+  current_version: latest_commit_hash,
```

Comment:
```suggestion
  currentVersion: latestCommitHash,
```

https://v2.vuejs.org/v2/style-guide/#Prop-name-casing-strongly-recommended